Building Applications with React and Flux.md

```diff
@@ -0,0 +1,15 @@
+# This is a development environment for React
+
+#### With React comming to Free Code camp soon. I thought this would be a aset to the community.
+
+ > This is designed to run in the cloud nine environment.
+> Once this is installed in a custom environment at cloud nine. All you have to do is type npm install from the bash terminal.
+> Then type gulp from the bash terminal.
+> This should set up a development environment. For you to work with React code.
```

Comment:
Your wiki is the authority, I would suggest "Your React development environment is now ready."